doc/nrf/drivers/wifi/nrf700x.rst

```diff
@@ -71,6 +71,14 @@ The nRF7002 DK uses QSPI whereas the nRF7002 EK uses SPI.
 
 To connect the nRF7002 EK to the SoC, the ``nrf7002ek`` shield is required.
 
+Kconfig configuration
+*********************
+
+The nRF700X Wi-Fi driver has the followi configurable options:
+
+.. options-from-kconfig::
```

Comment:
This file should be under `drivers/wifi/nrf700x` in case we add another Wi-Fi driver later.

Reply:
Done.